virttest/qemu_qtree.py

```diff
@@ -216,8 +217,11 @@ def generate_params(self):
             self.params['image_name'] = os.path.realpath(
                 self.block.get('backing_file'))
         elif self.block.get('file'):
-            self.params['image_name'] = os.path.realpath(
-                self.block.get('file'))
+            block_file = self.block.get('file')
+            image_name = os.path.realpath(block_file)
+            if not os.path.exists(block_file) and block_file.startswith('json:'):
```

Comment:
Looks only using `block_file.startswith('json:')` is enough.

Reply:
Thanks, have updated.